### PR TITLE
Relicense to MIT

### LINKED ISSUE #1: Choice of language server and license
Thanks @laughedelic for working on a Atom-IDE package for Scala.

I was wondering what motivated your choice for the underlying language server over other choices. In VSCode, we have language packages for Scala backed by `sbt`, `ensime` and `dotty` respectively. None uses `scalameta`, hence my surprise.

On a separate topic, I am not sure of the benefits of the `LGPL-3` license for Atom packages. Especially when the underlying backend (`scalameta`) is permissively licensed.

### PR DESCRIPTION
**Pull request by ghisvail**
As discussed in the issue tracker. Closes #1

### CODE REVIEW

**Comment by laughedelic** [comment]
Thanks @ghisvail! 
Could you also update the readme badge and rename the file to `LICENSE` (without `.md`)? It's not needed with the simple MIT license text and I discovered that Github recognizes license only if it's in the `LICENSE` file.

**Comment by laughedelic** [comment]
Also the reference in `package.json`

**Comment by ghisvail** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Atom IDE Scala language client
 
 [![](https://img.shields.io/github/release/laughedelic/atom-ide-scala/all.svg)](https://github.com/laughedelic/atom-ide-scala/releases/latest)
-[![](https://img.shields.io/badge/license-LGPLv3-blue.svg)](https://www.tldrlegal.com/l/lgpl-3.0)
+[![](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```
Comment (laughedelic) [inline README.md]: Why suddenly yellow? 😄 I don't have anything against, but I've never seen a yellow license badge..
Also, could you decapitalize `License`? For consistency with other badges.

**Comment by ghisvail** [comment]
No offense, but I really could not care less about cosmetic gimmicks such as badges (which I don't personally use). I googled "license badge MIT", used the first [link](https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba) and there we are.

Feel free to merge this PR and adapt the cosmetics to your likings afterwards.

**Comment by laughedelic** [comment]
Sure, no problem! Thanks for helping with this 👍

**Comment by ghisvail** [comment]
You're welcome. Thanks for your interest in integrating the Scala tooling for Atom.

**Comment by fommil** [comment]
It's a shame you changed the license... the MIT license gives to patent grant and allows proprietary forks. LGPL would have ensured that any distributions remained free (remaining fair to you), and that contributors cannot submit any code that they hold a patent right to, protecting all users.

**Comment by ghisvail** [comment]
Le 17 nov. 2017 19:59, "Sam Halliday" <notifications@github.com> a écrit :

It's a shame you changed the license... the MIT license gives to patent
grant and allows proprietary forks. LGPL would have ensured that any
distributions remained free (remaining fair to you), and that contributors
cannot submit any code that they hold a patent right to, protecting all
users.

On an Atom plugin? Come on...

I totally get it for a larger piece of software like Ensime. For an Atom
plugin however, it's not much justified.

**Comment by fommil** [comment]
If the software is too small to care, then.... why do you care enough to change it?

**Comment by laughedelic** [comment]
@fommil check our previous discussion here: https://github.com/laughedelic/atom-ide-scala/issues/1#issuecomment-343583697. I also checked repos in the ensime org before taking this decision and I saw that ensime-atom, ensime-vim use MIT (as well as the new ensime-lsp-clients), many other things use Apache-2.0. Same with most Scala.js projects I found. So I thought that it's better to stay aligned with the community and keep it simple for the potential contributors.

**Pull request merged**